### DISCUSSION TITLE
nice PrettyZoo

### DISCUSSION THREAD

**Discussion by hcqbuqingzhen** [vran-dev/PrettyZoo#187] [Show and tell]
你好，我寻找zk的gui过程中发现您的软件。
用起来挺好，所以打包到archlinux的aur上了。
https://aur.archlinux.org/packages/prettyzoo/

**Comment by vran-dev** [comment]
谢谢


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "vran-dev/PrettyZoo"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年6月14日(星期一) 凌晨1:11
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[vran-dev/PrettyZoo] nice PrettyZoo (#187)





 
你好，我寻找zk的gui过程中发现您的软件。
 用起来挺好，所以打包到archlinux的aur上了。
 https://aur.archlinux.org/packages/prettyzoo/
